jobs/payment-jobs/tasks/ejv_partner_distribution_task.py

```diff
@@ -161,23 +161,26 @@ def _create_ejv_file_for_partner(cls, batch_type: str):  # pylint:disable=too-ma
                 # JV Header
                 ejv_content = '{}{}'.format(ejv_content,  # pylint:disable=consider-using-f-string
                                             cls.get_jv_header(batch_type, cls.get_journal_batch_name(batch_number),
-                                                              journal_name, total))
+                                                            journal_name, total))
                 control_total += 1
 
                 line_number: int = 0
+                jv_invoice_order = []
```

Comment:
Move this up to where `invoice_id_list = []` is, that way when the header changes.. the sequence resets.

The header is based on the partner I believe?

Reply:
If you look above invoice_id_list, you'll see the headers.. headers are what we are basing the sequencing on. It should reset when a new header is introduced.